.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: python
 sudo: false
 python:
-    - '2.6'
     - '2.7'
```

Comment:
Mostly sigh at this not being Python 3. But I understand that it's too big. Not really a comment on this pull request. But a comment on the state of the universe.